cvat-ui/src/components/organization-page/top-bar.tsx

```diff
@@ -203,26 +235,24 @@ function OrganizationTopBar(props: Readonly<Props>): JSX.Element {
                             </span>
                         ) : (
                             <div ref={descriptionEditingRef}>
-                                <Input.TextArea
-                                    defaultValue={description}
-                                    onChange={(event: React.ChangeEvent<HTMLTextAreaElement>) => {
-                                        organizationDescription = event.target.value;
-                                    }}
-                                />
-                                <Button
-                                    className='cvat-submit-new-org-description-button'
-                                    size='small'
-                                    type='primary'
-                                    onClick={() => {
-                                        if (organizationDescription !== description) {
-                                            organizationInstance.description = organizationDescription;
-                                            dispatch(updateOrganizationAsync(organizationInstance));
-                                        }
-                                        setEditingDescription(false);
-                                    }}
+                                <Form
+                                    onFinish={onSubmitDescription}
+                                    form={descriptionForm}
+                                    initialValues={{ description }}
                                 >
-                                    Submit
-                                </Button>
+                                    <Form.Item name='description'>
+                                        <Input.TextArea />
```

Comment:
Maybe we should add validators here? I can enter any text in email and phone sections